build.gradle

```diff
@@ -108,6 +108,11 @@ tasks.register("updateCIBwcVersions") {
       ".buildkite/pipelines/periodic.bwc.template.yml",
       BuildParams.bwcVersions.allIndexCompatible
     )
+    writeBuildkiteList(
```

Comment:
I realize rewriting the existing file is a bit janky but I didn't want to do any major refactoring right now.